.github/workflows/linux-build-and-test.yaml

```diff
@@ -29,12 +29,15 @@ jobs:
     env:
       PG_SRC_DIR: pgbuild
       PG_INSTALL_DIR: postgresql
+      OPENSSL_ROOT_DIR: /usr/local/opt/openssl
```

Comment:
Is FindOpenSSL.cmake could not find this path by default?

Reply:
yes cmake won't find openssl without this

Reply:
Is it possible to add hints or something to the FindOpenSSL module?

Reply:
OPENSSL_ROOT_DIR is the way to pass hints but i moved it from env to argument